src/js/core/methods/helpers/__fixtures__/paramsValidator.js

```diff
@@ -362,6 +363,6 @@ export const getFirmwareRange = [
             { support: { trezor1: '1.6.2', trezor2: '2.1.0' }, shortcut: 'eth', type: 'ethereum' },
             DEFAULT_RANGE,
         ],
-        result: { '1': { min: '1.8.0', max: '0' }, '2': { min: '2.1.0', max: '0' } },
+        result: { '1': { min: '1.10.4', max: '0' }, '2': { min: '2.4.2', max: '0' } },
```

Comment:
This reverts the change made in https://github.com/trezor/connect/pull/928/commits/9fecde426e45dfb702f000902ce6544d5844f484